### PR TITLE
Lesson12

### PR DESCRIPTION
**Pull request by aksafan**

### CODE REVIEW

File: app/controllers/concerns/authentication_check.rb
Comment (gacurl) [inline app/controllers/concerns/authentication_check.rb:1]: Consider using `before_action` in the controller to ensure `is_user_logged_in` runs before actions that require authentication.

**Review by gacurl — APPROVED** [review]

File: app/controllers/concerns/authentication_check.rb
Comment (aksafan) [reply app/controllers/concerns/authentication_check.rb:1]: It is already used [here](https://github.com/aksafan/R7-rest/pull/1/files/1d0aa3cbac5ed2c9ab8bec17ae9aecdb7e4853a2#diff-01865460c66780c57ef1a6b264a8243df4f864f42ed25971888b239f3e98063fR4), and [here](https://github.com/aksafan/R7-rest/pull/1/files/1d0aa3cbac5ed2c9ab8bec17ae9aecdb7e4853a2#diff-01865460c66780c57ef1a6b264a8243df4f864f42ed25971888b239f3e98063fR4).

**Comment by gacurl** [comment]
I saw those. Excellent. And security throughout is crucial. No need to point out where you have the code. Crucial to cover all cases. Sent from iPhone. Please pardon brevity and typos.On Sep 7, 2024, at 6:31 PM, Anton Khainak ***@***.***> wrote:﻿
@aksafan commented on this pull request.



On app/controllers/concerns/authentication_check.rb:
It is already used here, and here.

**Comment by aksafan** [comment]
> I saw those. Excellent. And security throughout is crucial. No need to point out where you have the code. Crucial to cover all cases. Sent from iPhone. Please pardon brevity and typos.On Sep 7, 2024, at 6:31 PM, Anton Khainak ***@***.***> wrote:﻿ @aksafan commented on this pull request. On app/controllers/concerns/authentication_check.rb: It is already used here, and here. —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: ***@***.***>

Gotcha, thanks!

**Pull request merged**